backend/pkg/restaurant/store.go

```diff
@@ -19,3 +18,31 @@ func NewStore(db *sqlx.DB) core.RestaurantStore {
 func (s *restaurantStore) Exists(ctx context.Context, restaurantID int) (bool, error) {
 	return false, nil
 }
+
+func (s *restaurantStore) ListRestaurants(ctx context.Context) ([]*core.Restaurant, error) {
+	rows, err := s.db.QueryContext(ctx, `SELECT id, name, address FROM restaurants`)
+
```

Comment:
raczej nie zostawia się tu wolnego miejsca, żeby zaznaczyć, że sprawdzany error pochodzi z linijki powyżej :P ale to kosmetyka